.travis.yml

```diff
@@ -160,7 +160,7 @@ matrix:
     - name: "Deploy Pants PEX Unstable"
       os: linux
       language: python
-      stage: Deploy Pants Pex
+      stage: Deploy Pants Pex Unstable
```

Comment:
Using anchors for stage names really is the right thing to do, but lgtm.

Reply:
Ah I see. Thanks for the yaml trick! Adjusted.